array.go

```diff
@@ -746,6 +746,128 @@ func (a *Array) Equal(value interface{}) *Array {
 	return a.IsEqual(value)
 }
 
+// InList succeeds if array is listed by given [values...].
+// Before comparison, both array and value are converted to canonical form.
+//
+// values should be an array of slice of any type. This comparison adheres
+// element order.
+//
+// Example:
+//
+//	array := NewArray(t, []interface{}{"foo", 123})
+//	array.InList([]interface{}{"foo", 123})
+//
+//	array := NewArray(t, []interface{}{"foo", "bar"})
+//	array.InList([]string{}{"foo", "bar"})
+//
+//	array := NewArray(t, []interface{}{123, 456})
+//	array.InList([]int{}{123, 456})
```

Comment:
Let's provide example with multiple arguments to InList, to make it clear that it accepts list of slices, not a single slice.

Reply:
Addressed in `Array` and others as well.